grazel-gradle-plugin/src/test/kotlin/com/grab/grazel/gradle/variant/DefaultVariantBuilderTest.kt

```diff
@@ -153,7 +153,7 @@ class DefaultVariantBuilderTest {
     @Test
     fun `assert jvm variants are built`() {
         val variants = variantBuilder.build(jvmProject)
-        assertEquals(2, variants.size)
+        assertEquals(3, variants.size)
```

Comment:
Sorry I meant to add a new test case to that actually runs the `detekt` logic by setting it up in `VariantTestProject.kt` etc.